features/barprop.yml

```
@@ -0,0 +1,7 @@
+name: BarProp
+description: "BarProp is implemented by browser UI such as the address bar or scrollbars, historically to represent whether the user interface component was visible. For privacy reasons, the spec was changed to make `visible` represent whether is a popup or not. Apple has not yet implemented the change."
+spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#barprop
```

I think it's nicer to link to the prose before the IDL:

```suggestion
spec: https://html.spec.whatwg.org/multipage/nav-history-apis.html#browser-interface-elements
```